### ISSUE TITLE
IsAlive() should not count late activated units.

### ISSUE THREAD

**Issue by Prof-hilactic** [FlightControl-Master/MOOSE#97]
For the SPAWN function if we use late activation unit we cannot use the function IsAlive() to detect if the unit /group has spawned or not.

This is due to the way DCS work. A late activation unit is alive even if not activated yet

A work around at the moment is to use trigger.misc.getUserFlag

**Comment by FlightControl-Master** [comment]
Will fix this...

**Comment by Grey-Echo** [comment]
Is this still relevant? If not, please close this issue.

**Comment by Delta-99** [comment]
I think FC's change to IsAlive recently handles this case but will let him say for sure.

**Comment by Grey-Echo** [comment]
Ok, I tested this issue with this script : 

```lua
Tank = UNIT:FindByName("Unit #1")
BASE:E(Tank:IsAlive())

SCHEDULER:New(nil, function()
    SPAWN:New( "TankGroup" )
    BASE:E("SPAWNING !")
    end, {}, 5)

SCHEDULER:New(nil, function() BASE:E(Tank:IsAlive()) end, {}, 10)
```

And I got the following result : 
```
04312.715 INFO    SCRIPTING:      2(    -1)/E:                BASE00000.function(true)
04317.819 INFO    SCRIPTING:      5(    -1)/E:                BASE00000.function(SPAWNING !)
04322.820 INFO    SCRIPTING:      7(    -1)/E:                BASE00000.function(true)
04327.194 INFO    SCRIPTING:   4797(    13)/E:               EVENT00000.onEvent(S_EVENT_MISSION_END)
```

So I don't think this is fixed, sadly :(

**Comment by FlightControl-Master** [comment]
Who uses this and who is waiting for this? Testing IsAlive on an inactive group. This change can have big hidden implications. How to test if something is inactive?

**Comment by Delta-99** [comment]
Is this no good: http://wiki.hoggit.us/view/DCS_func_isActive

**Comment by FlightControl-Master** [comment]
Ok.i see! So testing of there is a unit and then also checking if it is active. Thanks!

**Comment by FlightControl-Master** [comment]
Working on it ..

**Comment by FlightControl-Master** [comment]
Fixed with pull request #363 . Please test.

**Comment by Grey-Echo** [comment]
Retested in Moose 20170327_1219 with the same code as above.
Works flawlessly.
Issue closed !
